site/src/components/Resources/AgentRowPreview.tsx

```diff
@@ -0,0 +1,159 @@
+import { makeStyles } from "@material-ui/core/styles"
+import { AppPreviewLink } from "components/AppLink/AppPreviewLink"
+import { FC } from "react"
+import { combineClasses } from "util/combineClasses"
+import { WorkspaceAgent } from "../../api/typesGenerated"
+import { Stack } from "../Stack/Stack"
+
+export interface AgentRowPreviewProps {
+  agent: WorkspaceAgent
+}
+
+export const AgentRowPreview: FC<AgentRowPreviewProps> = ({ agent }) => {
+  const styles = useStyles()
+
+  return (
+    <Stack
+      key={agent.id}
+      direction="row"
+      alignItems="center"
+      justifyContent="space-between"
+      className={styles.agentRow}
+    >
+      <Stack direction="row" alignItems="baseline">
+        <div className={styles.agentStatusWrapper}>
+          <div className={styles.agentStatusPreview}></div>
+        </div>
+        <Stack
+          alignItems="baseline"
+          direction="row"
+          spacing={4}
+          className={styles.agentData}
+        >
+          <Stack
+            direction="row"
+            alignItems="baseline"
+            spacing={1}
+            className={combineClasses([styles.noShrink, styles.agentDataItem])}
+          >
+            <span>Agent:</span>
+            <span className={styles.agentDataValue}>{agent.name}</span>
+          </Stack>
+
+          <Stack
+            direction="row"
+            alignItems="baseline"
+            spacing={1}
+            className={combineClasses([styles.noShrink, styles.agentDataItem])}
+          >
+            <span>OS:</span>
```

Comment:
And some translations in this file would be nice, too

Reply:
Sure thing!